src/components/Videos.vue

```diff
@@ -123,6 +124,51 @@ export default {
 			// reset and show poster after play
 			this.$el.autoplay = false
 			this.$el.load()
+		},
+
+		// Save video player's volume and mute status
+		saveVolume() {
+			var videoVolume = {
+				volume: this.$el.volume,
+				muted: this.$el.muted
+			}
+			// store volume status as data of the root component
+			this.$root.$data.videoVolume = videoVolume
```

Comment:
Ah, I just noticed you added this.
Usually it's not a good habit to use the $root handler if you can avoid it :)
Any particular reason you needed this?

Reply:
Sorry I didn't make it clear. I added it there because `$root` should be always available and persistent until the the page session gets closed, so now `localStorage` would only be necessary for across page sessions.

I did some reading and seems like the cases when modern browsers do not support `localStorage` or `sessionStorage` would most likely be browsing in private mode, so I think in this case storing the value in something similar to a global variable would be sufficient and therefore have it put in `$root`. 

Would you suggest a better idea? Otherwise it's probably fine to drop that part since it's for rare cases. 

Thanks.

Reply:
But $root refers to the app instance.
So it does not matter if you store the data from this.videoVolume ?
Also, since this will refer to the Videos component it make sense to store the data there, not in this.$root which is the global viewer root element and is irrelevant to those values :)

Reply:
Looks like the value gets lost once the viewer overlay is closed if it is stored in the Videos component. If we want to make that part useful, it should better be able to stay for longer. I used root mainly because it lives for the entire page session...

Reply:
But if we store it int he localstorage and is refreshed on each loadedMetadata event, how can it be lost? I think I'm missing something here :)
Or are you talking in case the browser does not support localstorage?

Reply:
Right. I'm talking about in the case when localStorage is not available. I added that when considering a fallback option.

Reply:
Ah, make more sense to me now! :)
I still don't like storing data that are not declared on the root, but it's fine I'd say!

Thanks for the explanation! :)

Reply:
Shall we put this in the catch then? as a real fallback? :)

Reply:
Sounds good. I'll do that.